project3/common/Roomba/Roomba.h

```diff
@@ -3,19 +3,31 @@
 
 #include <stdint.h>
 
+#define STRAIGHT 32768
+
 class Roomba {
   public:
     Roomba(uint8_t serial_connector, uint8_t brc_pin);
 
     bool init();
     void drive(int16_t velocity, int16_t radius);
+    void direct_drive(int16_t left_speed, int16_t right_speed);
+    void stop();
     void dock();
 
     bool check_oi_mode(uint16_t* mode);
     bool check_power(uint16_t* power);
     bool check_power_capacity(uint16_t* power);
+    bool check_light_bumper(uint16_t* data);
     void power_off();
 
+    bool check_virtual_wall();
+    bool check_left_bumper();
+    bool check_right_bumper();
```

Comment:
I'd like change the names of these functions, the check functions return boolean success status, but these return boolean data